agent/consul/fsm/fsm.go

```diff
@@ -40,6 +42,10 @@ func registerCommand(msg structs.MessageType, fn unboundCommand) {
 	commands[msg] = fn
 }
 
+func init() {
+	registerCommand(structs.ResourceOperationType, (*FSM).applyResourceOperation)
```

Comment:
is there a reason that this is here and not with the rest of the commands in the `init()` of commands_oss.go?